_members/cravey-luke.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Luke Cravey
 image: images/cravey-luke.jpg
-role: undergrad-student
+role: undergrad-alum
 links:
 email: tlc0053@auburn.edu
 ---
```

Comment:
Please update the description below as follows:

Luke worked in the lab an undergraduate researcher in Summer 2024 (with Mayukh) through the CASE REU. He modeled the thermodynamics and self-assembly of patchy colloids. Luke plans to pursue a Ph.D. in chemical engineering upon graduation.